Examples/ArcGISToolkitExamples/AppDelegate.swift

```diff
@@ -21,6 +21,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
         // Override point for customization after application launch.
+
+        #warning("Require user to sign in with an ArcGIS identity or set your developer API key")
+        /*
+         Use of Esri location services, including basemaps and geocoding, requires either an ArcGIS identity or an API Key. For more information see https://links.esri.com/arcgis-runtime-security-auth.
+         1) ArcGIS identity: An ArcGIS named user account that is a member of an organization in ArcGIS Online or ArcGIS Enterprise.
+         2) API key: A permanent key that gives your application access to Esri location services. Create a new API key or access existing API keys from your ArcGIS for Developers dashboard (https://links.esri.com/arcgis-api-keys).
+         Production deployment of applications built with ArcGIS Runtime requires you to license ArcGIS Runtime functionality. For more information see https://links.esri.com/arcgis-runtime-license-and-deploy.
+         */
+        // Uncomment the following line to access Esri location services using an API key.
+        // AGSArcGISRuntimeEnvironment.apiKey = “<#API Key#>”
```

Comment:
Looks like with the copy and paste of this doc different quotes were added. If you uncomment this line you will get an error saying, 
> Unicode curly quote found, replace with '"'

This should fix it when you delete those quotes and add new ones manually.

```suggestion
        // AGSArcGISRuntimeEnvironment.apiKey = "<#API Key#>"
```